### PR TITLE
feat: added PR template

### PR DESCRIPTION
**Pull request by dustytrinkets**
## What’s the focus of this PR
This PR adds this awesome PR template. Feel free to suggest modifications for it

## How to review this PR
With lots of love

Before submitting this PR, I made sure:

- [x] The code builds clean without any errors or warnings
- [x] I'm using our guidelines
- [ ] I've added unit tests if feasible

### CODE REVIEW

**Review by ardguezsoc — COMMENTED** [review]
I think that we dont need this because @inigomarquinez  wanted to prepare some templates for the organization One Beyond. So once it is done we would inherit that from there and use them by default because we are not using once. But im not sure ❓ ❓

**Comment by dustytrinkets** [comment]
> 

I wasn't aware of this, @inigomarquinez should we wait for the general one to be included?

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## What’s the focus of this PR
```
Comment (inigomarquinez) [inline pull_request_template.md:1]: Is it being added under the `.github` folder?

**Review by inigomarquinez — COMMENTED** [review]
Just one question about the location of the file

File: pull_request_template.md (hunk 1 above)
Comment (dustytrinkets) [reply pull_request_template.md:1]: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

apparently it is an option if we want it hidden, should I move it?

**Review by inigomarquinez — COMMENTED** [review]
We'll add one single template in the new organization. But as we don't know yet when that will be ready, we can proceed with this one and replace it with the new one once we migrate to the new organization. And that way you have this PR for the Hacktoberfest ;)

File: pull_request_template.md (hunk 1 above)
Comment (inigomarquinez) [reply pull_request_template.md:1]: Ohhhh, I didn't know that there were several options for that. Cool!!

In my opinion, it's better to have it hidden as it's not part as the code itself. But feel free to put it wherever you think is best. I'll approve the PR anyway.

**Review by inigomarquinez — APPROVED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (dustytrinkets) [reply pull_request_template.md]: Nice, let's move it then!

**Pull request merged**